bot.py

```diff
@@ -414,6 +418,22 @@ async def howmany(context: commands.Context) -> None:
 
     await context.channel.send(response)
 
+@client.command()
+async def help(context: commands.Context) -> None:
+    await context.send(help_message())
+
+@client.command()
+@commands.dm_only()
+async def status(context: commands.Context) -> None:
+    
+    week = compo.get_week(True)
+
+    for entry in week["entries"]:
+        if entry["discordID"] == context.author.id:
+            await context.send(entry_info_message(entry))
+            return
+    
+    await context.send(f"You haven't submitted anything yet! But if you want to you can with %ssubmit !" % client.command_prefix[0])
```

Comment:
Other than this, looks good to merge
```suggestion
    await context.send("You haven't submitted anything yet! But if you want to you can with %ssubmit !" % client.command_prefix[0])
```